README.md

```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
 * Contact: David Neto <dneto@google.com>
 * Compiler requirements: C++11 compatible compiler (MSVC 2013+ on Windows)
 
+## DXC
+DirectXShaderCompiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
+It can also compile HLSL into SPIR-V, contributed by Google.
+
+* Repository: https://github.com/Microsoft/DirectXShaderCompiler
+* License: University of Illinois Open Source License
+* Contact: Lei Zhang <antiagainst@google.com>
+* Platform: Windows
+* Compiler requirement: Visual Studio 2017
+* Download: https://khr.io/dxcappveyorbuild (rolling release)
```

Comment:
Would it be worth mentioning that **some** version is part of the Windows SDK?

Reply:
The version shipped in Windows SDK does not contain SPIR-V CodeGen. I think this doc is regarding Vulkan ecosystem; so I'm not sure it matters to mention that version here.